src/Control/dropDown.component.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const DropDown = (props) => {
+    const {id, value, onChange, option} = props;
+            
+    return (
+        <select id={id} value={value} onChange={onChange}>        
+           {
+             option.length !== 0 ? option.map(year =>(<option value={year}>{year}</option>)) :
```

Comment:
I didnt remember seeing this option tag in wbs, change it and sorry I didnt mention it in wbs that you are using year inside map which is specific, change to data